spec/graph_ql/query_spec.rb

```diff
@@ -50,6 +50,26 @@
       assert_equal(GraphQL::Nodes::FragmentDefinition, query.fragments['cheeseFields'].class)
     end
 
+    describe 'scalar coercion' do
```

Comment:
Could you create  `spec/graph_ql/types/id_type_spec.rb` to test this behavior? Fine with me to just copy-paste the other `let`s etc. 

Even though `Query` is one of the hotspots, I'm hoping to keep the spec pretty slim and test other behaviors in specific places!